src/Nethermind/Nethermind.Ssz/SszStream.cs

```diff
@@ -0,0 +1,77 @@
+//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Runtime.CompilerServices;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Nethermind.Ssz
+{
+    public struct SszStream
+    {
+        public byte[] Data { get; private set; }
+
+        public int Position { get; set; }
+        
+        public SszStream(int length)
+        {
+            Data = new byte[length];
+            Position = 0;
+        }
+        
+        public static void Encode(byte value)
```

Comment:
why all this static notimplemented methods?

Reply:
A placeholder for stream serialization, possibly not needed, need to check in WIP thins, otherwise we would have them on a bramch for a long time